### ISSUE TITLE
addAuthorNameToInboxNotifications doesn't work on localized sites

### ISSUE THREAD

**Issue by unarist** [soscripted/sox#242]
### Current Behaviour

addAuthorNameToInboxNotifications feature doesn't work on locilized sites because they localized inbox too.

For example (Stack Overflow in Japanese):

- comment: コメント
- chat reply: チャットの返信
- answer: 回答
- edit suggested: 編集が提案されました

and different sites (ru/pt/es) have different translations.

Could you detect notification type by other elements (e.g. link URL)?

---

### SOX Errors logged to the browser console -- F12 (if available)

```
(unknown) SOX:  SOX does not currently support get author information for type: チャットの返信
(unknown) SOX:  SOX does not currently support get author information for type: コメント
(unknown) SOX:  SOX does not currently support get author information for type: 回答
(unknown) SOX:  SOX does not currently support get author information for type: 編集が提案されました
```

---

### Features Enabled 

    ["Appearance-addAuthorNameToInboxNotifications","Appearance-alignBadgesByClass","Appearance-answerTagsSearch","Appearance-bulletReplace","Appearance-colorAnswerer","Appearance-dragBounty","Appearance-fixedTopbar","Appearance-highlightQuestions","Appearance-isQuestionHot","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-scrollToTop","Appearance-spoilerTip","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-showCommentScores","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-enhancedEditor","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Editing-downvotedPostsEditAlert","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideHireMe","Sidebar-hideHotNetworkQuestions","Sidebar-linkedToFrom","Sidebar-hotNetworkQuestionsFiltering","Chat-chatEasyAccess","Chat-replyToOwnChatMessages","Chat-renameChat","Voting-betterCSS","Voting-grayOutVotes","Voting-stickyVoteButtons","Voting-disableOwnPostVoteButtons","Unsorted-alwaysShowImageUploadLinkBox","Unsorted-linkedPostsInline","Unsorted-parseCrossSiteLinks","Unsorted-quickAuthorInfo","Unsorted-shareLinksMarkdown","Unsorted-sortByBountyAmount","Unsorted-warnNotLoggedIn"]

**Comment by shu8** [comment]
@unarist thanks for posting this! :) just to clarify -- does the feature not work entirely, or is this talking about the messages that appear in the console? (and you can see the author name in the inbox for eg. comments?)

**Comment by unarist** [comment]
@shu8 not work entirely. I can see the author name on English sites, but cannot see on the Japanese site.

**Comment by shu8** [comment]
@unarist thanks! Sorry for delay again, but this should be fixed in [dev 2.0.21](https://github.com/soscripted/sox/raw/dev/sox.user.js)! :).

Please could you install it using the above link (no need to remove the version you have installed), and confirm whether it works for you? :)

**Comment by unarist** [comment]
@shu8 I've tested dev 2.0.21, but it didn't work.

```js
switch (type) { 
    case 'comment' || link.indexOf('posts/comments/') > -1:
        // ...
        break;
}
```

It seems to `'comment' || link.indexOf('posts/comments/') > -1` is always evaluated to `'comment'`.

**Comment by shu8** [comment]
hmm ok, I'll make an account on one of those sites and try and fix it ASAP :)

**Comment by shu8** [comment]
@unarist Thanks for the patience! Should be fixed (for real) in [dev 2.0.23](https://github.com/soscripted/sox/raw/dev/sox.user.js) :)

**Comment by unarist** [comment]
@shu8 Thanks, it works nice!
